packages/mui-base/src/InputUnstyled/InputUnstyled.spec.tsx

```diff
@@ -23,7 +23,7 @@ const InputInput = React.forwardRef(function InputInput(
 
 const styledInput = <InputUnstyled components={{ Root: InputRoot, Input: InputInput }} />;
 
-const PolymorphicComponentTest = () => {
+const polymorphicComponentTest = () => {
```

Comment:
The idea here is that `PolymorphicComponentTest` looks like a component (and therefore would allow a hooks call). But inside of it we're declaring new components which is bad. Since it's not actually a component, we adjust the name to reflect that.